zkevm-circuits/src/state_circuit/multiple_precision_integer.rs

```diff
@@ -1,32 +1,48 @@
 use super::{lookups, param::*};
 use crate::util::Expr;
-use eth_types::{Address, Field};
+use eth_types::{Address, Field, ToLittleEndian, Word};
 use halo2_proofs::{
     circuit::{Layouter, Region, Value},
     plonk::{Advice, Column, ConstraintSystem, Error, Expression, Fixed, VirtualCells},
     poly::Rotation,
 };
 use itertools::Itertools;
-use std::marker::PhantomData;
+use std::{cmp::min, marker::PhantomData};
 
 pub trait ToLimbs<const N: usize> {
     fn to_limbs(&self) -> [u16; N];
+    fn annotation() -> &'static str;
 }
 
 impl ToLimbs<N_LIMBS_ACCOUNT_ADDRESS> for Address {
-    fn to_limbs(&self) -> [u16; 10] {
+    fn to_limbs(&self) -> [u16; N_LIMBS_ACCOUNT_ADDRESS] {
         // address bytes are be.... maybe just have everything be later?
         // you will need this in the future later because it makes the key ordering more
         // obvious
         let le_bytes: Vec<_> = self.0.iter().rev().cloned().collect();
         le_bytes_to_limbs(&le_bytes).try_into().unwrap()
     }
+    fn annotation() -> &'static str {
```

Comment:
just a qn: for this kind of constant retrieval, should we add `#[inline(always)]` for it